### DISCUSSION TITLE
Multiple Plex servers?

### DISCUSSION THREAD

**Discussion by funnelcloudservices** [meisnate12/Plex-Meta-Manager#447] [General]
I have multiple Plex servers.  Can this be used on multiple ones?  I know it says "# Can be individually specified per library as well" but I'm not entirely sure if that means what I think it means.

**Comment by meisnate12** [comment]
There's an example [here](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Libraries-Attributes#library-name)

**Reply by funnelcloudservices** [reply]
thank you so much. I totally missed that section.  

Great script btw.  incredibly helpful.
